packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3152,6 +3169,47 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   }
 
+  async seekRow(inputString: string, isBackwards = false): Promise<void> {
+    const {
+      gotoValueSelectedColumn: selectedColumn,
+      gotoValueSelectedFilter,
+      lastSeekedRow,
+    } = this.state;
+    const { model } = this.props;
+    if (isIrisGridProxyModel(model)) {
+      const { table } = model;
```

Comment:
We should define `seekRow` as a function in `IrisGridModel` so this functionality can be abstracted out. We generally shouldn't call `table` functions directly from anything except within `IrisGridModel`.

Reply:
To be clear, add an `abstract seekRow` method and `get isSeekRowAvailable()` method to `IrisGridModel` (similar to `abstract export` and `get isExportAvailable`), implement `seekRow` in `get isSeekRowAvailable` in `IrisGridTableModel` (where `isSeekRowAvailable` returns `this.table.seekRow != null`).